go.sum

```diff
@@ -1,20 +1,5 @@
 github.com/BurntSushi/toml v0.3.1 h1:WXkYYl6Yr3qBf1K79EBnL4mak0OimBfB0XUf9Vl28OQ=
```

Comment:
Usually if go.sum changed, go.mod is changed too. Check if you commited both files.